spec/models/articles_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'validations' do
+    before { subject.valid? }
+    describe 'title' do
+      context 'when blank' do
+        subject { build(:article, title: nil) }
+        it { expect(subject.errors[:title]).to include("can't be blank") }
```

Comment:
I recommend following solution:
```ruby
it do
  is_expected.to be_invalid
  expect(subject.errors[:title]).to include("can't be blank")
end
```

Motivation - if model will be valid we would like to know about it before testing any errors inclusion due to it does not make sense